python/tank/commands/setup_project.py

```diff
@@ -104,20 +113,21 @@ def __init__(self):
         
         # Special setting used by the shotgun desktop app to handle the current form of distributed
         # configs
-        self.parameters["auto_path"] = { "description": ("Expert setting. Setting this to true means that a blank "
-                                                         "path entry is written to the shotgun site pipeline "
-                                                         "configuration. This can be used in conjunction with "
-                                                         "a localized core to create a site configuration which "
-                                                         "can have different locations on different machines. It "
-                                                         "is then up to the bootstrap logic of the code that "
-                                                         "starts up toolkit to determine where to go look for the "
-                                                         "configuration. When setting this to true, you typically "
-                                                         "only need to specify the path to the current operating "
-                                                         "system configuration."),
-                                     "default": False,
-                                     "type": "bool" }
-        
-        
+        self.parameters["auto_path"] = {
+            "description": "Deprecated. Do not use this! --- "
```

Comment:
Maybe we should also mention this is only used with the Shotgun Desktop. centralized configs without a project are ignored by the resolver.

Reply:
i added a comment!